apps/web/modules/settings/organizations/new/_components/OnboardMembersView.tsx

```diff
@@ -34,41 +34,39 @@ const useOrgCreation = () => {
   const session = useSession();
   const utils = trpc.useUtils();
   const [serverErrorMessage, setServerErrorMessage] = useState("");
-  const { useOnboardingStore, isBillingEnabled } = useOnboarding();
+  const { useOnboardingStore } = useOnboarding();
   const { reset } = useOnboardingStore();
 
-  const checkoutMutation = trpc.viewer.organizations.createWithPaymentIntent.useMutation({
-    onSuccess: (data) => {
+  // Single mutation for all flows (billing, self-hosted, admin)
+  const intentToCreateOrgMutation = trpc.viewer.organizations.intentToCreateOrg.useMutation({
```

Comment:
We call this on the last step - no matter if its billing enabed etc. 

Onboarding Intent gets created here if you are a user or an admin targeting themselves.